test/Hydration-test.js

```diff
@@ -54,7 +59,7 @@ describe('Hydration', () => {
 
   it('should calculate the average ounces per day', () => {
     let sumOunces = repoData.getAverageDailyOunces();
-    expect(sumOunces).to.equal(67);
```

Comment:
Updates reviewed; will continue to update sleep tests to reflect new methods needed.